recipes/manormfast/build.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-
-$PYTHON setup.py install 
```

Comment:
Is this tool deprecated? Should we keep the recipe?

Reply:
fair enough, will still keep it in the channel for rep. reasons.

Reply:
Thanks!

Reply:
As explained by @zfrenchee, the newer version just fixes everything up and changes the misleading name to a proper one. Sorry about that!